golang/lib/binding_constructors/binding_constructors.go

```diff
@@ -127,14 +123,14 @@ func NewLoadStaticFilesResponse(copiedstaticFileRelativeFilepaths map[string]str
 //                                        Start Service
 // ==============================================================================================
 func NewStartServiceArgs(
-		serviceId string,
-		image string,
-		usedPorts map[string]bool,
-		entrypointArgs []string,
-		cmdArgs []string,
-		envVars map[string]string,
-		enclaveDataVolMntDirpath string,
-		filesArtifactMountDirpaths map[string]string) *kurtosis_core_rpc_api_bindings.StartServiceArgs {
+	serviceId string,
```

Comment:
This extra indentation is intentional, so that the arguments aren't at the same level as the rest of the code

Reply:
It's automatically generated when I use the command `Code > Reformat Code` in Go Land.
I'll move it as it was before.